src/range.rs

```diff
@@ -185,9 +185,10 @@ indexed_range_impl!{i32}
 indexed_range_impl!{isize}
 
 // other Range<T> with just Iterator
-unindexed_range_impl!{u64}
-unindexed_range_impl!{i64}
-
+unindexed_range_impl!{u64, u64}
+unindexed_range_impl!{i64, u64}
+#[cfg(has_i128)] unindexed_range_impl !{u128, u128}
+#[cfg(has_i128)] unindexed_range_impl !{i128, u128}
```

Comment:
I'm surprised this parsed correctly with the space before `!`, but I'm still going to tweak that so syntax highlighting is correct too.